frontend/app/components/account-form.js

```diff
@@ -65,15 +86,24 @@ export default class AccountForm extends BaseFormComponent {
   }
 
   @action
-  setSelectedTeam(team) {
-    this.selectedTeam = team;
-    if (this.changeset.group !== null && this.changeset.group.id !== undefined)
-      this.changeset.group = null;
+  setSelectedTeam(selectedTeam) {
+    this.selectedTeam = selectedTeam;
+    this.changeset.team = selectedTeam;
+    this.changeset.group = null;
+    this.selectedGroup = null;
+
+    this.store.query("group", { team_id: this.selectedTeam.id }).then(groups => {
+      this.availableGroups = groups;
+
+      // groups[0] gives undefined, WHAT IS GROUPS?
+      this.setGroup(groups.filter(group => true || group)[0]);
```

Comment:
I'd like to return the first element of groups. `this.setGroup(groups[0])` doesn't work though, as groups is not an array. The filter function returns an array, so it does work when filtering nothing. The `true || group` is here, such that ember-test doesn't complain about the unused variable `group`. Please, if anyone knows, provide a better solution.